src/DataManagement/Entries/Entry.cs

```diff
@@ -17,7 +17,17 @@ public class Entry : IEntry
         public int Importance { get; set; }
 
 
-        public Entry(string title, decimal amount,  DateTime date, bool recurring, int importance)
+        public static Entry CreateLocalEntry(string title, decimal amount, DateTime date, bool recurring, int importance)
+        {
+            return new Entry(title, amount, date, recurring, importance);
+        }
+
+        public static Entry CreateLocalEntry(IGoal goal, DateTime date, bool recurring, int importance)
+        {
+            return new Entry(goal.Title, goal.Amount, date, recurring, importance);
+        }
+
```

Comment:
This one is nice

Reply:
Functionally it's exactly the same thing, but I noticed that is pretty confusing sometimes, why we don't need an ID and user ID to call a constructor, and sometimes we do, so this is just purely to make the code less confusing

Reply:
I we're speaking on this level, I can see why this is an improvement. And as I can see you've already changed code in front, so changes should not cause any problems. Nice.